src/views/ImportFlow.vue

```diff
@@ -285,5 +285,13 @@ export default ImportFlow
   overflow: hidden;
   white-space: nowrap;
 }
+</style>
+<style lang="scss" scoped>
+//The bootstrap default is what the design calls for but seems to get overridden somewhere else I can't find in order to style the block selection banner
+.alert-primary {
```

Comment:
You're right about the `primary` colour we adapted it with Viamo colour but I don't know who made the change.
@jcbashdown , I'd suggest we use `alert-info` here instead of primary, that way we don't need to override the style.

Reply:
It's a bit off from the design but I'm not sure Amen will mind and it's much cleaner. I'll make the change